pkgs/top-level/coq-packages.nix

```diff
@@ -39,19 +42,22 @@ let
       iris = callPackage ../development/coq-modules/iris {};
       ltac2 = callPackage ../development/coq-modules/ltac2 {};
       math-classes = callPackage ../development/coq-modules/math-classes { };
-      inherit (callPackage ../development/coq-modules/mathcomp {})
-        mathcomp_ mathcomp-config
```

Comment:
Is there any advice for users of the `mathcomp_` function in order to write code compatible with revisions of nixpkgs before and after this refactoring?

Reply:
I think making an override using `mathcomp.override {version = mathcomp-config;}` should have roughly the same outcome, except fot the syntax change `owner/branch` to `owner:branch`.

Reply:
Were should I document this?

Reply:
Good question… Maybe just a comment.
(might be the case that all users of that attribute have already read this thread…)

Reply:
I added a comment here: https://github.com/NixOS/nixpkgs/blob/b0bd1aae6163b85989b8028b25b058fdb1eed9ab/pkgs/development/coq-modules/mathcomp/default.nix#L11